src/ducks/categories/CategoriesHeader.jsx

```diff
@@ -36,9 +36,22 @@ const stTableCategory = catStyles['bnk-table-category']
 const IncomeToggle = ({ withIncome, onToggle }) => {
   const theme = useCozyTheme()
   const { t } = useI18n()
+
+  const handleClick = useCallback(
+    ev => {
+      return onToggle(ev.target.checked)
+    },
+    [onToggle]
+  )
   return (
     <div className={cx(styles.CategoriesHeader__Toggle, styles[theme])}>
-      <Toggle id="withIncome" checked={withIncome} onToggle={onToggle} />
+      <Switch
+        id="withIncome"
+        disableRipple
```

Comment:
I'm surprised we need to specify that, isn't there a global option to disable the ripple effects?

Reply:
I haven't seen one yet.

Reply:
<img width="812" alt="Capture d’écran 2020-07-20 à 17 24 46" src="https://user-images.githubusercontent.com/1107936/87955395-fc580000-caad-11ea-9b1c-11b64f2fd934.png">

I guess we can do the same with the Switch component (https://material-ui.com/getting-started/faq/)

Reply:
Nice, thanks !